source/MaterialXCore/Look.cpp

```diff
@@ -162,15 +162,15 @@ vector<VariantAssignPtr> MaterialAssign::getActiveVariantAssigns() const
 // Lookgroup methods
 //
 
-LookVec LookGroup::getActiveLooks() const
+LookVec LookGroup::getAllEnabledLooks() const
```

Comment:
Wasn't sure what to call this because getEnabledLooks was taken, open to suggested names.